benchmark/process/coverage.js

```diff
@@ -0,0 +1,130 @@
+// This benchmark is meant to exercise a grab bag of code paths that would
+// be expected to run slower under coverage. To use this benchmark, create an
+// alternate node bin that enables coverage and compare it against ./node
+//
+// ./node-coverage
+// #!/bin/bash
+// NODE_V8_COVERAGE=./coverage ./node $@
```

Comment:
What is this part of the comment trying to communicate?

Reply:
You could copy paste this, and use it as the bin to benchmark, alternatively I could try to use child_process perhaps and enable the environment variable programmatically.